src/contracts/interfaces/GPv2ERC1271.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+library ERC1271 {
+    /// @dev Value returned by a call to `isValidSignature` if the signature
+    /// was verified successfully. The value is defined in the EIP-1271 standard
+    /// as: bytes4(keccak256("isValidSignature(bytes32,bytes)")
+    bytes4 internal constant MAGICVALUE = 0x1626ba7e;
```

Comment:
It can actually be derived as a `constant` (no immutable needed) as Solidity has support for const-exprs. However, I prefer to leave it this way as:
1. This is how it is defined in the [specification](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1271.md#specification)
2. `immutable` and `constants` from expressions aren't supported in assembly (the latter being a Solidity limitation).